patches/minecraft/net/minecraft/pathfinding/PathNavigator.java.patch

```diff
@@ -0,0 +1,14 @@
+--- a/net/minecraft/pathfinding/PathNavigator.java
++++ b/net/minecraft/pathfinding/PathNavigator.java
+@@ -208,9 +208,9 @@
+       Vector3d vector3d = this.func_75502_i();
+       this.field_188561_o = this.field_75515_a.func_213311_cf() > 0.75F ? this.field_75515_a.func_213311_cf() / 2.0F : 0.75F - this.field_75515_a.func_213311_cf() / 2.0F;
+       Vector3i vector3i = this.field_75514_c.func_242948_g();
+-      double d0 = Math.abs(this.field_75515_a.func_226277_ct_() - ((double)vector3i.func_177958_n() + 0.5D));
++      double d0 = Math.abs(this.field_75515_a.func_226277_ct_() - ((double)vector3i.func_177958_n() + (this.field_75515_a.func_213311_cf() + 1) / 2D));
```

Comment:
This line, and the other changed line should both have a comment at the end of them along the lines of `//Forge: Fix MC-94054`

Reply:
Ah right right, forgot to re-add that, fixed.

Reply:
No, add them on the same line to keep the patch size down, and add it to both of them